Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM mhart/alpine-node:6
 
-MAINTAINER tkqubo
+MAINTAINER kyleholzinger
```

Comment:
The `MAINTAINER` instruction is now deprecated by Docker. In version `v0.3.1` of the Code Climate spec, we recommend using the `LABEL` instruction, also recommended by Docker:

```
LABEL maintainer "Kyle Holzinger <youremail@example.com>"
```